app/views/decidim/sabarca/shared/_user_group.html.erb

```diff
@@ -2,13 +2,13 @@
   <div class="card">
     <div class="card__content">
       <h5 class="card__title">
-        <%= user_group.url.present? ? (link_to user_group.name, user_group.url, target: "_blank") : user_group.name %>
+        <%= link_to user_group.name, user_group.url, target: "_blank" %>
```

Comment:
Why changing this line?

Reply:
I thought url was mandatory, but it was not:

https://github.com/ajuntament-sabarca/decidim-sabarca/blob/update_to_decidim_0_13/app/forms/decidim/sabarca/registration_form.rb

I've refactored the code. Thanks